src/test/plugin.test.ts

```diff
@@ -59,6 +60,40 @@ test("infer.ts", () => {
     expect(schemaDeclaration.toStringHierarchy()).toBe(
         "Variable abc:ZodObject<Abc>",
     );
+
+    expect(
+        (typeDeclaration.type as ReflectionType)!.declaration!.getChildByName(
+            "def",
+        )!.flags.isOptional,
+    ).toBe(false);
```

Comment:
I'm sure you have a better way to do this. 😄

Reply:
I have some utilities to in typedoc, but they aren't exported, this works